docs/configuration.asciidoc

```diff
@@ -184,10 +182,10 @@ options are:
 * `"debug"`
 * `"trace"` (sets `logging.log_level` to 5)
 
-Options are case-insensitive.
+Options are not case sensitive.
```

Comment:
Not case sensitive instead of case insensitive, right?

Reply:
```suggestion
Options are case-insensitive
```